Gemfile

```diff
@@ -28,7 +28,7 @@ group :backend, :frontend, :core, :api do
   gem 'database_cleaner', '~> 1.3', require: false
   gem 'factory_bot_rails', '~> 4.8', require: false
   gem 'rspec-activemodel-mocks', '~> 1.1', require: false
-  gem 'rspec-rails', '~> 3.7', require: false
+  gem 'rspec-rails', '~> 4.0.0.beta2', require: false
```

Comment:
@ericsaupe yes it is, see https://github.com/solidusio/solidus/pull/3236#issuecomment-520341615.

Reply:
@ericsaupe yeah, I think so. Since this will be released with 2.10 (I guess) there's no rush anyway, I imagine a stable release is behind the corner for RSpec.

Reply:
I don’t see why we need to wait until a stable rspec 4 has been released. We don’t ship this to any of our users but solely use this Gemfile for testing. So, IMO its totally fine.

Reply:
@tvdeyen I don't think there's a specific reason, it's just that there's no rush to merge this IMO and using the stable now will ensure we don't run into any unexpected issues down the road when the stable is released. I'm fine with both courses of action though, we can either wait for stable or merge with the beta.

Reply:
I see lets leave it as is and merge this PR so people can start testing master with Rails 6